building/build-debs/homeworld-admin-tools/src/authority.py

```diff
@@ -23,7 +23,24 @@ def name_for_decrypted_file(name_of_encrypted_file):
     if name_of_encrypted_file.endswith(ENCRYPTED_EXTENSION):
         return name_of_encrypted_file[:-len(ENCRYPTED_EXTENSION)]
     raise ValueError("Filename " + name_of_encrypted_file + " does not have expected suffix '" + ENCRYPTED_EXTENSION + "'.")
-    
+   
+def validate_pem_file(dir_path: str, file_name: str) -> None:
+    full_file_name = os.path.join(dir_path, file_name)
+    with open(full_file_name) as pem_file:
+        has_certificate = False
+        for line in pem_file:
+            # PEM data that is a certificate should have the following header
+            if line.startswith("-----BEGIN CERTIFICATE-----"):
+                if has_certificate:
+                    command.fail("The PEM file \"" + file_name + "\"should only contain 1 certificate, but it contains additional certificates!")
+                has_certificate = True
+            # The file should not have other types of PEM data
+            elif line.startswith("-----BEGIN "):
+                command.fail("The PEM file \"" + file_name + "\"should only contain 1 certificate, but it contains additional PEM data!")
```

Comment:
Error message is unclear. Saying "should only contain 1" implies that you've already found one, even though this can occur when that's not the case.